app/src/main/java/com/google/android/stardroid/source/proto/ProtobufAstronomicalSource.java

```diff
@@ -108,7 +108,7 @@ public GeocentricCoordinates getSearchLocation() {
     ArrayList<TextSource> points = new ArrayList<TextSource>(proto.getLabelCount());
     for (LabelElementProto element : proto.getLabelList()) {
       points.add(new TextSourceImpl(getCoords(element.getLocation()),
-          resources.getString(element.getStringIndex()),
+          "TestElement",
```

Comment:
Why this change?

Reply:
It can be seen that further iterating through the implementation ```resource``` variable we find that it is an empty collection declared [here](https://github.com/sky-map-team/stardroid/blob/4ec2f8327e4d2ee99493d238bea9c9cb73520b56/app/src/main/java/com/google/android/stardroid/source/proto/SourceProto.java#L2282). Also errors were popping up when building without this fix. Please correct me if I am wrong.